diff -ruN cdf37_1-dist/Makefile patched/Makefile
--- ./Makefile	2018-12-17 08:46:06.000000000 -0600
+++ ./Makefile	2020-04-06 18:35:05.000000000 -0500
@@ -260,10 +260,10 @@
 
 CURSES_macosx_gnu=yes
 CC_macosx_gnu=gcc
-LD_macosx_gnu=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool -macosx_version_min 10.6
+LD_macosx_gnu=libtool -macosx_version_min 10.6
 LDOPTIONS_macosx_gnu=-dynamic
 PIC_macosx_gnu=
-COPTIONS_macosx_gnu=-m64 -D__MACH__ -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -O2
+COPTIONS_macosx_gnu=-mmacosx-version-min=10.6 -m64 -D__MACH__ -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -O2
 COPTIONSZlib_macosx_gnu=-DHAVE_STDARG_H -DHAVE_UNISTD_H
 COPTIONSld_macosx_gnu=
 FOPTIONS_macosx_gnu=-w $(MACOSxFopt)
